### DISCUSSION TITLE
Probe appears as not active

### DISCUSSION THREAD

**Discussion by gatekeeper8** [nzymedefense/nzyme#364] [General]
Dears,
Following documentation I've just configured nzyme. 
On dashboard I've found that probe is not working. Red exlamation mark "NOT ACTIVE! Check nzyme logs."
When I looked at nzyme.log I've found:

`05:53:36.819 [main] INFO  horse.wtf.nzyme.systemstatus.SystemStatus - Set system status [RUNNING].
05:53:36.819 [main] INFO  horse.wtf.nzyme.systemstatus.SystemStatus - Set system status [TRAINING].
05:53:37.717 [main] INFO  horse.wtf.nzyme.NzymeLeaderImpl - Training period ends in <300> seconds.
05:53:37.718 [main] INFO  horse.wtf.nzyme.NzymeLeaderImpl - Initializing nzyme version: 1.0.0 built at [2021-03-20T23:19:33Z].
05:53:37.718 [main] INFO  horse.wtf.nzyme.NzymeLeaderImpl - Active alerts: [UNEXPECTED_BSSID, UNEXPECTED_SSID, CRYPTO_CHANGE, UNEXPECTED_CHANNEL, UNEXPECTED_FINGERPRINT, BEACON_RATE_ANOMALY, MULTIPLE_SIGNAL_TRACKS, PWNAGOTCHI_ADVERTISEMENT, BANDIT_CONTACT]
05:53:37.718 [main] INFO  horse.wtf.nzyme.ouis.OUIManager - Fetching and updating list of OUIs from [http://standards-oui.ieee.org/oui/oui.txt]. This might take a moment.
05:53:52.426 [main] INFO  horse.wtf.nzyme.ouis.OUIManager - Done! Now <29754> OUIs in memory. Download time <855ms>, parsing time <13s>.
05:53:52.443 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [OUIUpdater] for every <12 HOURS> with <12 HOURS> initial delay.
05:53:52.455 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [MeasurementsWriter] for every <1 MINUTES> with <1 MINUTES> initial delay.
05:53:52.455 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [MeasurementsCleaner] for every <10 MINUTES> with <0 MINUTES> initial delay.
05:53:52.456 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [BeaconRateWriter] for every <60 SECONDS> with <60 SECONDS> initial delay.
05:53:52.457 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [BeaconRateCleaner] for every <10 MINUTES> with <0 MINUTES> initial delay.
05:53:52.458 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [SignalIndexHistogramWriter] for every <60 SECONDS> with <60 SECONDS> initial delay.
05:53:52.458 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [SignalIndexHistogramCleaner] for every <10 MINUTES> with <0 MINUTES> initial delay.
05:53:52.578 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [VersioncheckThread] for every <60 MINUTES> with <0 MINUTES> initial delay.
05:53:52.579 [periodicals-2] INFO  horse.wtf.nzyme.periodicals.versioncheck.VersioncheckThread - Starting to check for most recent nzyme version.
05:53:52.602 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [BeaconRateAnomalyAlertMonitor] for every <60 SECONDS> with <60 SECONDS> initial delay.
05:53:52.602 [main] INFO  horse.wtf.nzyme.periodicals.PeriodicalManager - Scheduling [SignalTrackMonitor] for every <60 SECONDS> with <60 SECONDS> initial delay.
05:53:53.547 [main] INFO  horse.wtf.nzyme.NzymeLeaderImpl - Started web interface and REST API at [http://192.168.90.233:22900/]. Access it at: [http://192.168.90.233:22900/]
05:53:53.607 [probe-loop-0] INFO  horse.wtf.nzyme.dot11.probes.Dot11Probe - Commencing 802.11 frame processing on [wlx9cefd5fae138] ... (⌐■_■)–︻╦╤─ – – pew pew
05:53:53.667 [periodicals-2] INFO  horse.wtf.nzyme.periodicals.versioncheck.VersioncheckThread - Successfully completed version check. Everything seems up to date.
05:53:53.733 [probe-loop-0] INFO  horse.wtf.nzyme.dot11.probes.Dot11Probe - Building PCAP handle on interface [wlx9cefd5fae138]
05:53:57.379 [grizzly-http-server-0] WARN  horse.wtf.nzyme.rest.resources.authentication.AuthenticationResource - Failed login attempt for user [admin].
05:53:59.850 [probe-loop-0] INFO  horse.wtf.nzyme.dot11.probes.Dot11Probe - PCAP handle for [broad-monitor-wlx9cefd5fae138] acquired. Cycling through channels <1,2,3,4,5,6,7,8,9,10,11>.
05:56:02.565 [grizzly-http-server-0] INFO  horse.wtf.nzyme.rest.resources.authentication.AuthenticationResource - Creating session for user [admin]
05:58:37.718 [pool-8-thread-1] INFO  horse.wtf.nzyme.NzymeLeaderImpl - Training period is over!
05:58:37.720 [pool-8-thread-1] INFO  horse.wtf.nzyme.systemstatus.SystemStatus - Unset system status [TRAINING].
`
In mean time at syslog:

`Apr  8 05:53:59 nzyme kernel: [  966.513191] IPv6: ADDRCONF(NETDEV_CHANGE): wlx9cefd5fae138: link becomes ready
Apr  8 05:53:59 nzyme systemd-networkd[645]: wlx9cefd5fae138: Gained carrier
Apr  8 05:53:59 nzyme kernel: [  966.519460] device wlx9cefd5fae138 entered promiscuous mode
`
And what I've understood it looks good. 
Could You share an idea what to check more and how to fix it ?

gatekeeper

**Comment by gatekeeper8** [comment]
Update from my side:
seems like there is a problem with a wifi dongle. 
I've got two : TPLINK TL-WN821N which works flawlessly on 2.4GHz also with nzyme.
and Pandawireless PAU07, which works flawlessly everywhere but not with nzyme. 
Any hints from Your side ?

**Reply by gatekeeper8** [reply]
Looks like talking to myself ;) 
I've solved a problem which I created myself. 
Switching from Virtualbox to baremetal solved problem with Panda. 
I hope that my story will help someone.

**Reply by lennartkoopmann** [reply]
Hi! I was just about to look into it. Glad you got it figured out already! :)
